winml/lib/Api.Experimental/LearningModelJoinOptions.cpp

```diff
@@ -0,0 +1,51 @@
+#include "lib/Api.Experimental/pch/pch.h"
+#include "LearningModelJoinOptions.h"
+
+namespace WINML_EXPERIMENTALP {
+
+LearningModelJoinOptions::LearningModelJoinOptions()
+{
+  GUID guid;
```

Comment:
why not only do this if you need to instead of every time, even if a manual override is provided?

Reply:
The manual override is provided after construction, i could add another constructor here, but thought it wasnt needed for an experimental api.

Reply:
or set it to null, and when you need to use it initialize it if it the caller didn't provide one.

Reply:
but then multiple chained models would have the same prefix and have potential name collisions.